### ISSUE TITLE
SSH option

### ISSUE THREAD

**Issue by ElisaRojas** [fp7-netide/IDE#28]
Hi @cstritzke ,

As you know, we need to use the IDE for our use case, but, unfortunately, I still can't find the way to run it... If I run the Vagrant option, I have the problems in #18  and #27 . 

If I choose SSH instead, I cannot connect to my VM (it's the VM I have been using for all demos, developments, etc. and I know it's working correctly). I'm attaching a screenshot of my config, just in case something is wrong. 

![ssh](https://cloud.githubusercontent.com/assets/2779496/18307507/1c090e4c-74f1-11e6-8b7a-7a4d1b3cbc8e.png)

Basically, the SSH option seems to run, but if I press "core on", nothing appears. Also, I'm a bit concerned about the location of components, how can I specify where the core, etc. is installed?

Sorry for being such a pain :( But we need this working asap to continue working on UC2.

Thank you!

**Comment by cstritzke** [comment]
Yes, the location of the components is quite restrictive at the moment I'm afraid. If the Core is not at $HOME/netide/core-karaf, it won't start.

**Comment by cstritzke** [comment]
Also, is your ssh port for the VM really 2222?

**Comment by ElisaRojas** [comment]
Haha, I used 2222 because Gonzalo sent me a valid example with that value... but 22 makes much more sense. In any case, I have the same result with 22 :(

Could you provide some guide specifying the _**needed**_ location of all components? In case the Vagrant option does not work, I'll need to create a VM from scratch with those locations. (I'm going to try now the VM created with Vagrant via SSH in the meantime).

Thank you!

**Comment by cstritzke** [comment]
Core: ~/netide/core-karaf
Mininet Configurations: ~/netide/mn-configs
ODL Shim: ~/netide/distribution-karaf-0.4.0-Beryllium/
Engine Repo: ~/netide/Engine
Apps: ~/netide/apps 

Please let me know if you need more.

**Comment by ElisaRojas** [comment]
Core: ~/netide/core-karaf is new? I mean, I saw the others correctly, but the core was not there, but in ~/apache... (I have the previous commit of master, not the current one).

**Comment by cstritzke** [comment]
Yep, I changed it for the sake of consistency. Sorry, that's pretty new. Should have notified you.

**Comment by ElisaRojas** [comment]
One question, would it be possible to change some of those folders? Mainly the core and ODL shim folders, we can adapt the others in our VMs.

The reason is that currently the people running demos (Raúl, Sergio with the testbed, Juanma, me). We all have different locations (we all have the same, but different than yours) and using yours would mean to reinstall many of the stuff in our VMs, which is a pain :( I think the SSH option is crucial for testing the UCs and we would need it working. What do you think? Let's talk about it now...

**Comment by ElisaRojas** [comment]
Hi again, I'm copying my current VM (30 mins remaining) and I will modify it to have those specific folders. I'll let you know if it works, in around 1h and then we can decide whether to change some of them or not.

**Comment by ElisaRojas** [comment]
I'm closing the issue since the customized folders have been added in the ssh_config_extension branch. Thanks!
